packages/schema/lib/functional-constraints/AuthFieldisSafe.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const jsonschema = require('jsonschema');
+
+const AUTH_FIELD_ID = '/AuthFieldSchema';
+const AUTH_FIELDS_ID = '/AuthFieldsSchema';
+
+const FORBIDDEN_KEYS = [
+  'access_token',
```

Comment:
Just like the different versions of api_key, perhaps, we should do the same for access_token, refresh_token, and set-cookie as follows:

- access_token
- access-token
- accesstoken
- refresh_token
- refresh-token
- refreshtoken
- set-cookie
- set_cookie
- setcookie